index.js

```diff
@@ -48,6 +56,19 @@ var cacheControlNoTransformRegExp = /(?:^|,)\s*?no-transform\s*?(?:,|$)/
 function compression (options) {
   var opts = options || {}
 
+  if (hasBrotliSupport) {
+    // set the default level to a reasonable value with balanced speed/ratio
+    if (opts.params === undefined) {
+      opts = objectAssign({}, opts)
+      opts.params = {}
+    }
+
+    if (opts.params[zlib.constants.BROTLI_PARAM_QUALITY] === undefined) {
+      opts.params = objectAssign({}, opts.params)
+      opts.params[zlib.constants.BROTLI_PARAM_QUALITY] = 4
+    }
```

Comment:
```suggestion
    opts.params = objectAssign({ [zlib.constants.BROTLI_PARAM_QUALITY]: 4 }, opts.params)
```

Technically different, but would allow setting to `undefined` to use the default node value.

Reply:
good catch

Reply:
It doesn't work in older versions of Node.js :c